Sources/NineSunsSDK/NSEncryption.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  NSEncryption.swift
+//  NineSunsSDK
+//
+//
+import Foundation
+import CryptoKit
+import TweetNacl
+
+public struct NSEncryption {
+  
+  private static func encryptAsymmetric(secretOrSharedKey: Data, message: String) -> String? {
+    // Convert message to Data
+    guard let messageData = message.data(using: .utf8) else {
+      print("Failed to convert message to Data")
+      return nil
+    }
+    
+    // Generate a nonce
+    let nonceData = Data((0..<NACL_NONCE_LENGTH).map { _ in UInt8.random(in: 0...255) })
+    
+    do {
+      let encryptedMessage = try NaclSecretBox.secretBox(message: messageData, nonce: nonceData, key: secretOrSharedKey)
+      
+      // Concatenate nonce and encrypted message
+      let fullMessage = nonceData + encryptedMessage
+      
+      // Encode the result as base64
+      let encodedMessage = fullMessage.base64EncodedString()
+      
+      return encodedMessage
+    } catch {
+      print("Encryption failed: \(error)")
+      return nil
+    }
+  }
+  
+  /**
+   * Asymmetric encryption method for string.
+   * - Parameters:
+   *   - myPrivateKey: User's private encryption key.
+   *   - theirPublicKey: Recipient's public key.
```

Comment:
Update the description for myPrivateKey and theirPublicKey a

Reply:
Thank em, I updated it